interface/wx/cmndata.h

```diff
@@ -535,6 +547,18 @@ class wxPrintDialogData : public wxObject
     */
     bool GetSelection() const;
 
+    /**
+        Returns @true if the user requested that the current page be printed.
+
+        Note that the "current page" is a defined by the application.
```

Comment:
```suggestion
        Note that the "current page" is defined by the application.
```
Or, a noun should be added after the "a".